tern/classes/image.py

```diff
@@ -19,7 +19,7 @@ class Image(object):
         get_layer_diff_ids: returns a list of layer diff ids only
         to_dict: return a python dictionary representation of the image
     '''
-    def __init__(self, id=None):
+    def __init__(self, id=None):  # pylint: disable=redefined-builtin
```

Comment:
I am really interested in knowing why pylint picks this up as a redefined-builtin while it doesn't pick it up for the Command class that also has the same pattern: `def __init__(self, shell_command)`

Reply:
"id" is the built-in here:  ```id(object) -> integer``` Return the identity of an object.